cashu/mint/startup.py

```diff
@@ -88,7 +88,10 @@ async def rotate_keys(n_seconds=60):
 
 
 async def start_mint_init():
-    await migrate_databases(ledger.db, migrations)
-    await ledger.startup_ledger()
-    logger.info("Mint started.")
+    try:
+        await migrate_databases(ledger.db, migrations)
+        await ledger.startup_ledger()
+        logger.info("Mint started.")
+    except:
```

Comment:
Could you also print the Exception `e`, and use `logger.error()`?
```suggestion
    except Exception as e
```